lib/ui/dart_runtime_hooks.cc

```diff
@@ -199,6 +200,24 @@ void Logger_PrintString(Dart_NativeArguments args) {
   }
 }
 
+void SaveCompilationTrace(Dart_NativeArguments args) {
```

Comment:
Did we want to expose this to all users?

It needs documentation explaining how and when to use it.